torchbenchmark/util/model.py

```diff
@@ -60,8 +60,10 @@ def enable_profiling_executor():
         torch._C._get_graph_executor_optimize(graph_executor)
 
 class BenchmarkModel(metaclass=PostInitProcessor):
-    DEFAULT_TRAIN_BSIZE: Optional[int] = None
-    DEFAULT_EVAL_BSIZE: Optional[int] = None
+    # May be overwritten by individual models
+    DEFAULT_TRAIN_BSIZE = 4
```

Comment:
Sorry but I am kinda concerned about doing this since many models do not support customizing batch sizes and setting it to 4 will mislead users.

I am thinking we only set `DEFAULT_TRAIN_BSIZE` for `hf_Whisper`